assets/main.js

```diff
@@ -26,3 +31,56 @@ document.addEventListener('DOMContentLoaded', () => {
         }
     });
 });
+
+function startEventSource() {
+	if (queryStreamEvent) return;
+
+	if (!window.location.pathname.startsWith("/querylog")) return;
+
+	const query_logs = document.getElementById("query-logs");
+
+	queryStreamEvent = new EventSource(`/api/queries`);
+
+	queryStreamEvent.onmessage = ({ data }) => {
+		try {
+		    const payload = JSON.parse(data);
+                    let row = query_logs.insertRow(1);
```

Comment:
we should eventually remove elements to not exceed (let's say) 50 queries... otherwise the site becomes a bit unusable...

Reply:
done in 1f708f7, settled for 100.